packages/synapse-react-client/src/components/SynapseTable/SynapseTableRenderers.tsx

```diff
@@ -130,14 +134,41 @@ export const directDownloadColumn = columnHelper.display({
   },
 })
 
+const getEntityOrRowId = (
```

Comment:
These two functions, getEntityOrRowId() and getEntityOrRowVersion() are the meat of the PR.  Get the column index from the table meta.  If defined, then use that value from the Row.values[].   Otherwise return what it used to (the rowId or versionNumber?.toString()).

Reply:
They are pretty self explanatory but a bit of context for each in a function doc wouldn't hurt--I'm finding the more custom configuration we add to Synapse Tables UI, the harder it is to remember how everything works!

Reply:
Done